### PR TITLE
Regressed dep and fixed other

### PR DESCRIPTION
**Pull request by mbround18**
# Description

## Contributions

- Dependency hell rework for a `clap` which was failing on beta. 
- Reworked based on Clippy feedback

## Checklist

- [x] I added one or multiple labels which best describes this PR.
- [x] I have tested the changes locally.
- [x] This PR has a reviewer on it.
- [ ] I have validated my changes in a docker container and on Ubuntu. (Only needed for Odin or Docker Changes)

### CODE REVIEW

File: Makefile.toml [hunk 1]
```diff
@@ -63,7 +63,7 @@ dependencies = ["member_format", "member_clippy", "docker:build"]
 [tasks.access]
 workspace = false
 command = "docker-compose"
-args = ["-f", "./docker-compose.dev.yml", "exec", "--user", "steam", "valheim", "bash"]
+args = ["-f", ".`/docker-compose.dev.yml", "exec", "--user", "steam", "valheim", "bash"]
```
Comment (LovecraftianHorror) [inline Makefile.toml:66]: (away from computer so cant test manually) Is this change intentional?

**Review by LovecraftianHorror — COMMENTED** [review]
Left a comment on the only thing that stood out to me

So there were dependency issues with the new `clap` beta?

File: Makefile.toml (hunk 1 above)
Comment (mbround18) [reply Makefile.toml:66]: lol rogue back tic :P ima fix that

**Comment by mbround18** [comment]
> 
> 
> Left a comment on the only thing that stood out to me
> 
> So there were dependency issues with the new `clap` beta?

Yeah, clap beta 4 currently screams about a missing readme doc in their repo. its not something I can fix sadly :<

**Comment by LovecraftianHorror** [comment]
Ah that's rough. Nothing stuck out to me in the issue tracker about what was up, but I guess it's a beta for a reason

At least the change back to `clap v2` was mostly painless
